dist/bootstrap-slider.js

```diff
@@ -1376,8 +1376,8 @@ function _typeof(obj) { return obj && typeof Symbol !== "undefined" && obj.const
 				var val = this._calculateValue(true);
```

Comment:
this is a generated file. please make your change in the `/src` file.

Also...where does this `_snapToValue()` function come from? I dont see it defined anywhere in our source code